### ISSUE TITLE
client-core has dependencies on social, should only be other way around

### ISSUE THREAD

**Issue by shawwwwwwww** [XRFoundation/XREngine#2836]
![image](https://user-images.githubusercontent.com/18633264/129714410-d131549b-7cc8-4aaa-8412-0cf331fa1214.png)

**Comment by HexaField** [comment]
If you're referring to the social package, this is the social folder in client-core, so this seems like a non issue?

**Comment by NPatel10** [comment]
I agree with @HexaField. I double checked all the dependencies and turns out they are from social folder in client-core package not from social package

**Comment by shawwwwwwww** [comment]
![image](https://user-images.githubusercontent.com/18633264/130169399-a895764c-c04e-4df4-a4e6-ab3a9f3a7eea.png)

Turns out you are *mostly* right but there is one :D
